pkg/middleware/validation.go

```diff
@@ -15,7 +15,7 @@ import (
 // NewValidationMiddleware builds a middleware function that performs validation checks by calling
 // an external webhook
 func NewValidationMiddleware(entry log.Entry, validatorHook string) buffalo.MiddlewareFunc {
-	const op errors.Op = "actions.ValidationMiddleware"
+	const op errors.Op = "actions.NewValidationMiddleware"
```

Comment:
This can be NewValidationMiddleware. If you are doing that can you add a new errors.Op in the middleware function that is returned with `const op errors.Op = "actions.ValidationMiddleware` and update the span function with the correct op

Reply:
yep! Sure